src/components/FontAwesomeIcon.js

```diff
@@ -1,8 +1,11 @@
-import { parse as faParse, icon as faIcon } from '@fortawesome/fontawesome-svg-core'
-import { defineComponent, computed, watch } from 'vue'
+import { classList, objectWithKey } from '../utils'
+import { computed, defineComponent, watch } from 'vue'
+import { ICON_PACKS_STARTING_VERSION, SVG_CORE_VERSION } from '../utils'
```

Comment:
I added this import here and the "semver" import. 
The other imports were previously added, I just alphabetized them.